src/main.js

```diff
@@ -7,6 +7,16 @@
     const URLSearchParams = window.URLSearchParams;
     // do not enable the plugin for old browsers
     if (!fetch || !Request || !URLSearchParams) return;
+	
+    // poly fill for startsWith
+    if (!String.prototype.startsWith) {
```

Comment:
Bad idea, this project is not about polyfills.

Reply:
Figured I should have it in the event you were making the script as backwards compatible as possible. Arguably there's an additional amount of work being spent here because indexOf is going to walk through the entire string to find "http", but obviously urls aren't particularly big.